zerver/lib/test_helpers.py

```diff
@@ -598,3 +601,98 @@ def zulip_reaction_info() -> Dict[str, str]:
         emoji_code='zulip',
         reaction_type='zulip_extra_emoji',
     )
+
+class SpamInTestOutputException(Exception):
+    pass
+
+class SpamFinder:
```

Comment:
This bundle probably better belongs in `test_runner.py` than `test_helpers.py` -- the latter is for things we might import from an individual test.

Reply:
Actually, let's put this new code in a new file, `zerver/lib/test_console_output.py` ; I think that'll be happier.  (I'd misunderstood it as being called not from `test_classes.py`).